tests/framework/Optimizers/GeneticAlgorithms/discrete/unconstrained/testGAMinwRep.xml

```diff
@@ -92,9 +92,9 @@
       <variable name="x3">
         <distribution>uniform_dist_Repl_1</distribution>
       </variable>
-
       <objective>ans</objective>
       <TargetEvaluation class="DataObjects" type="PointSet">optOut</TargetEvaluation>
+      <constant name="unused">1.23456789</constant>
```

Comment:
please update the TestInfo regarding the changes.

Reply:
added